js/functions.js

```diff
@@ -0,0 +1,29 @@
+function checkLength (inputString, maxLength){
+  return inputString.length <= maxLength;
+}
+console.log(checkLength('проверяемая строка', 20));
+
+const checkPalindrome = function (string) {
+  string = string.replaceAll(' ', '');
+  const normString = string.toLowerCase();
+  let newString = '';
+  for (let i = (normString.length - 1); i >= 0; i--){
+    newString += normString[i];
+  }
+  return normString === newString;
+};
+console.log(checkPalindrome('Лёша на полке клопа нашёл'));
+
+const getNumbers = (string) => {
+  let finalNumber = '';
+  for (let i = 0; i <= string.length; i++) {
+    let sepIndex = parseInt(string[i], 10);
+    if (Number.isNaN(sepIndex)){
```

Comment:
```suggestion
    if (Number.isNaN(sepIndex)) {
```

Всегда нужно следить за «чистотой» кода и его стилизации, если перед `{` ставишь пробел в одном месте, значит его нужно ставить везде в таких случаях. Есть много инструментов, которые помогают это делать более-менее автоматически. Но на начальных этапах не стоит ими сильно увлекаться, потому что необходимо выработать привычку контролировать свой код.